### PR TITLE
Hk fix html select

### PR DESCRIPTION
**Pull request by hanneskaeufler**
As shown in #288 and #286, the select/options is not quite working right.

- [x] Fix weird bug. Why is Volvo showing up twice???
- [x] We are only `returning` the html in order to assert in the tests because we have no way to test what `@view` contains. Is that nice?

### CODE REVIEW

File: spec/lucky/select_helpers_spec.cr [hunk 1]
```diff
@@ -5,6 +5,21 @@ private class TestPage
 
   def render
   end
+
+  def select(field)
```
Comment (paulcsmith) [inline spec/lucky/select_helpers_spec.cr]: Maybe `render_select`?

File: spec/lucky/select_helpers_spec.cr [hunk 2]
```diff
@@ -5,6 +5,21 @@ private class TestPage
...
+    select_input field do
+    end
+    self
+  end
+
+  def options(field, options)
```
Comment (paulcsmith) [inline spec/lucky/select_helpers_spec.cr]: And `render_options`? That way it's clear that it isn't using the `options` and `select` tags

File: spec/lucky/select_helpers_spec.cr [hunk 3]
```diff
@@ -21,20 +36,22 @@ end
...
+    <option value="2">Volvo</option><option value="3">BMW</option>
     HTML
   end
 
   it "renders selected option" do
-    (view.options_for_select form.company_id, [{"Volvo", 1}]).to_s.should contain <<-HTML
-    <option value="1" selected="true">Volvo</option>
+    view.options(form.company_id, [{"Volvo", 1}, {"BMW", 3}])
+      .html.to_s.should contain <<-HTML
+    <option value="1" selected="true">Volvo</option><option value="3">BMW</option>
```
Comment (paulcsmith) [inline spec/lucky/select_helpers_spec.cr]: I think this should also have another assertion that checks that `BMW` does not contain a `selected` attribute

Comment (hanneskaeufler) [reply spec/lucky/select_helpers_spec.cr]: That should be covered by asserting the `<option value="3">BMW</option>` shouldn't it? Nevertheless I changed the assertions from `containt` to `eq` as I dont think we want anything else in there anyway.

**Comment by hanneskaeufler** [comment]
Thanks paul, I pushed one more commit concerning your notes.

File: spec/lucky/select_helpers_spec.cr (hunk 3 above)
Comment (paulcsmith) [reply spec/lucky/select_helpers_spec.cr]: Oh you're right. I didn't see that last option. This looks great. And I like the change to `eq` 👍

**Comment by paulcsmith** [comment]
Closes #288 
Closes #286

**Pull request merged**